framework/scripts/agent_upgrade.py

```diff
@@ -148,13 +148,13 @@ def main():
 
     arg_parser = argparse.ArgumentParser()
     arg_parser.add_argument("-a", "--agent", type=str, help="Agent ID to upgrade.")
-    arg_parser.add_argument("-r", "--repository", type=str, help="Specify a repository URL.")
+    arg_parser.add_argument("-r", "--repository", type=str, help="Specify a repository URL. [Default: {0}]".format(common.wpk_repo_url))
```

Comment:
`argparse` uses the [`default` parameter](https://docs.python.org/2.7/library/argparse.html#default) to set default values. I think we should use it.

Reply:
I thought it did both but I was wrong. Thanks @vikman90 😄 !